workspace/lib/lita/handlers/jenkin.rb

```diff
@@ -0,0 +1,48 @@
+require 'jenkins_api_client'
+require 'jenkinhelper/jenkin'
+module Lita
+  module Handlers
+    class Jenkin < Handler
+      config :ip, required: true, type: String
+      config :username, required: true, type: String, default: ''
+      config :password, required: true, type: String, default: ''
+
+      include ::JenkinHelper::Jenkin
+
+      route(
+         /.*build_jenkin(.*)/,
+         :build_job_now,
+         command: false,
+         help: {t('help.build_jenkin.syntax') => t('help.build_jenkin.desc')}
+      )
+
+      def build_job_now(response)
+        argument = response.matches[0][0].strip
+        job_name, params = argument.split(' ')
+        response.reply(t('build_jenkin.building', job_name: job_name))
+        begin
+          if job_exists?(job_name)
+            if (params.nil? or params.empty?)
+              client.api_post_request("/job/#{path_encode job_name}/build", {}, true)
+            else
+              client.api_post_request("/job/#{path_encode job_name}/buildWithParameters", params, true)
+            end
+            queue_result = get_item_in_queue(job_name)
+            unless queue_result == true
```

Comment:
I think that <code>== true</code> is not necessary

Reply:
it should be == false instead of true. I mistook this line.